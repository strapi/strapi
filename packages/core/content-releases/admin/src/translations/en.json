{
  "content-manager-edit-view.add-to-release.select-label": "Select a release",
  "content-manager-edit-view.add-to-release.select-placeholder": "Select",
  "content-manager-edit-view.add-to-release.action-type-label": "What do you want to do with this entry?",
  "content-manager-edit-view.add-to-release.cancel-button": "Cancel",
  "content-manager-edit-view.add-to-release.continue-button": "Continue",
  "content-manager-edit-view.add-to-release": "Add to release",
  "content-manager-edit-view.add-to-release.notification.success": "Entry added to release",
  "content-manager-edit-view.add-to-release.no-releases-message": "No available releases. Open the list of releases and create a new one from there.",
  "content-manager-edit-view.add-to-release.redirect-button": "Open the list of releases",
  "content-manager-edit-view.list-releases.title": "{isPublish, select, true {Will be published in} other {Will be unpublished in}}",
  "content-manager-edit-view.remove-from-release": "Remove from release",
  "content-manager-edit-view.scheduled.date": "{date} at {time} ({offset})",
  "content-manager-edit-view.edit-release": "Edit release",
  "content-releases.content-manager-edit-view.edit-entry": "Edit entry",
  "content-manager-edit-view.remove-from-release.notification.success": "Entry removed from release",
  "content-manager-edit-view.release-action-menu": "Release action options",
  "content-manager.list-view.releases.header": "To be released in",
  "content-manager-list-view.add-to-release": "Add to release",
  "content-manager-list-view.add-to-release.cancel-button": "Cancel",
  "content-manager-list-view.add-to-release.continue-button": "Continue",
  "content-manager-list-view.add-to-release.select-label": "Select a release",
  "content-manager-list-view.add-to-release.select-placeholder": "Select",
  "content-manager-list-view.add-to-release.action-type-label": "What do you want to do with these entries?",
  "content-manager-list-view.add-to-release.notification.success.title": "Successfully added to release.",
  "content-manager-list-view.add-to-release.notification.success.message": "{entriesAlreadyInRelease} out of {totalEntries} entries were already in the release.",
  "content-manager.notification.entry-error": "Failed to get entry data",
  "content-manager.list-view.releases-number": "{number} {number, plural, one {release} other {releases}}",
  "plugin.name": "Releases",
  "pages.Releases.title": "Releases",
  "pages.Releases.header-subtitle": "Create and manage content updates",
  "pages.Releases.max-limit-reached.title": "You have reached the {number} pending {number, plural, one {release} other {releases}} limit.",
  "pages.Releases.max-limit-reached.message": "Upgrade to manage an unlimited number of releases.",
  "pages.Releases.max-limit-reached.action": "Explore plans",
  "pages.PurchaseRelease.subTitle": "Manage content updates and releases.",
  "pages.PurchaseRelease.not-available": "Releases is only available as part of a paid plan. Upgrade to create and manage releases.",
  "header.actions.add-release": "New Release",
  "header.actions.refresh": "Refresh",
  "header.actions.publish": "Publish",
  "header.actions.open-release-actions": "Release edit and delete menu",
  "header.actions.edit": "Edit",
  "header.actions.delete": "Delete",
  "header.actions.created": "Created",
  "header.actions.created.description": "{hasCreatedByUser, select, true { by {createdBy}} other { by deleted user}}",
  "modal.release-created-notification-success": "Release created",
  "modal.release-updated-notification-success": "Release updated",
  "modal.title": "{isCreatingRelease, select, true {New release} other {Edit release}}",
  "modal.form.input.label.release-name": "Name",
  "modal.form.input.label.schedule-release": "Schedule release",
  "modal.form.input.label.date": "Date",
  "modal.form.input.label.time": "Time",
  "modal.form.input.label.timezone": "Timezone",
  "modal.form.input.clearLabel": "Clear",
  "modal.form.button.submit": "{isCreatingRelease, select, true {Continue} other {Save}}",
  "modal.form.time.has-passed": "Selected time has already passed.",
  "pages.Details.header-subtitle": "{number, plural, =0 {No entries} one {# entry} other {# entries}}",
  "pages.Releases.tab-group.label": "Releases list",
  "pages.Releases.tab.pending": "Pending ({count})",
  "pages.Releases.tab.done": "Done",
  "page.Releases.tab.emptyEntries": "No releases",
  "pages.Details.tab.emptyEntries": "This release is empty. Open the Content Manager, select an entry and add it to the release.",
  "page.ReleaseDetails.table.header.label.name": "name",
  "page.ReleaseDetails.table.header.label.locale": "locale",
  "page.ReleaseDetails.table.header.label.content-type": "content-type",
  "page.ReleaseDetails.table.header.label.action": "action",
  "content-releases.page.ReleaseDetails.table.header.label.status": "status",
  "page.ReleaseDetails.table.action-published": "This entry was <b>{isPublish, select, true {published} other {unpublished}}</b>.",
  "pages.ReleaseDetails.publish-notification-success": "Release was published successfully.",
  "dialog.confirmation-message": "Are you sure you want to delete this release?",
  "page.Details.button.openContentManager": "Open the Content Manager",
  "pages.Releases.notification.error.title": "Your request could not be processed.",
  "pages.Releases.notification.error.message": "Please try again or open another release.",
  "pages.Releases.not-scheduled": "Not scheduled",
  "pages.ReleaseDetails.groupBy.label": "Group by {groupBy}",
  "pages.ReleaseDetails.groupBy.aria-label": "Group by",
  "pages.ReleaseDetails.entry-validation.already-published": "Already published",
  "pages.ReleaseDetails.entry-validation.ready-to-publish": "Ready to publish",
  "pages.ReleaseDetails.entry-validation.modified": "Ready to publish changes",
  "pages.ReleaseDetails.entry-validation.already-unpublished": "Already unpublished",
  "pages.ReleaseDetails.entry-validation.ready-to-unpublish": "Ready to unpublish",
  "pages.ReleaseDetails.entry-validation.not-ready": "Not ready to publish",
  "pages.ReleaseDetails.groupBy.option.content-type": "Content-Types",
  "pages.ReleaseDetails.groupBy.option.locales": "Locales",
  "pages.ReleaseDetails.groupBy.option.actions": "Actions",
  "pages.ReleaseDetails.header-subtitle.scheduled": "Scheduled for {date} at {time} ({offset})",
  "pages.ReleaseDetails.entry-validation.fields": "Fields",
  "pages.Settings.releases.title": "Releases",
  "pages.Settings.releases.description": "Create and manage content updates",
  "pages.Settings.releases.timezone.label": "Default timezone",
  "pages.Settings.releases.setting.default-timezone-notification-success": "Default timezone updated.",
  "pages.ReleaseDetails.entry-validation.fields.error": "{errors} errors on fields.",
  "pages.ReleaseDetails.entry-validation.fields.success": "All fields are filled correctly.",
  "pages.ReleaseDetails.entry-validation.fields.see-errors": "See errors",
  "pages.ReleaseDetails.entry-validation.review-stage.not-enabled": "This entry is not associated to any workflow.",
  "pages.ReleaseDetails.entry-validation.review-stage.not-ready": "This entry is not at the required stage for publishing. ({stageName})",
  "pages.ReleaseDetails.entry-validation.review-stage.ready": "This entry is at the required stage for publishing. ({stageName})",
  "pages.ReleaseDetails.entry-validation.review-stage.stage-not-required": "No required stage for publication."
}
