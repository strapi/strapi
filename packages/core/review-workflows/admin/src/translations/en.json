{
  "settings.page.title": "Review Workflows",
  "settings.page.subtitle": "{count, plural, one {# stage} other {# stages}}",
  "settings.page.isLoading": "Workflow is loading",
  "settings.page.delete.confirm.body": "All entries assigned to deleted stages will be moved to the previous stage. Are you sure you want to save?",
  "settings.stage.name.label": "Stage name",
  "settings.not-available": "Review Workflows is only available as part of the Enterprise Edition. Upgrade to create and manage workflows.",
  "settings.review-workflows.workflow.stageRequiredToPublish.label": "Required stage for publishing",
  "settings.review-workflows.workflow.stageRequiredToPublish.any": "Any stage",
  "settings.review-workflows.workflow.stageRequiredToPublish.hint": "Prevents entries from being published if they are not at the required stage."
}
