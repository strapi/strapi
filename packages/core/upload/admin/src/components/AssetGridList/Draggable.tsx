import * as React from 'react';

import { useDrag, useDrop } from 'react-dnd';

interface DraggableProps {
  id: string | number;
  index: number;
  children: React.ReactNode;
  moveItem: (fromIndex: number, toIndex: number) => void;
}

export const Draggable = ({ children, id, index, moveItem }: DraggableProps) => {
  const ref = React.useRef(null);

  const [, drop] = useDrop({
    accept: 'draggable',
    hover(hoveredOverItem: { id: number; index: number }) {
      if (!ref.current) {
        return;
      }

      if (hoveredOverItem.id !== id) {
        moveItem(hoveredOverItem.index, index);

        hoveredOverItem.index = index;
      }
    },
  });

  const [{ isDragging }, drag] = useDrag({
    type: 'draggable',
    item() {
      return { index, id };
    },
    collect: (monitor) => ({
      isDragging: monitor.isDragging(),
    }),
  });

  const opacity = isDragging ? 0.2 : 1;

  drag(drop(ref));

  return (
    <div ref={ref} style={{ opacity, cursor: 'move' }}>
      {children}
    </div>
  );
};
