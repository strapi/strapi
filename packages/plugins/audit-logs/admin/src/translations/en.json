{
  "plugin.name": "Audit Logs",
  "plugin.description": "Automated audit logging for all content changes"
}

