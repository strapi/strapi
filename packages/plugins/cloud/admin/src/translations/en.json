{
  "plugin.name": "Deploy",
  "Homepage.title": "Fully-managed Cloud Hosting for your Strapi Project",
  "Homepage.subTitle": "Follow this 2 steps process to get Everything You Need to Run Strapi in Production.",
  "Homepage.githubBox.title.versioned": "Project pushed to GitHub",
  "Homepage.githubBox.title.not-versioned": "Push your project on GitHub",
  "Homepage.githubBox.subTitle.versioned": "You did it! You're just one step ahead of having your project hosted online.",
  "Homepage.githubBox.subTitle.not-versioned": "Your project has to be versioned on GitHub before deploying on Strapi Cloud.",
  "Homepage.githubBox.buttonText": "Upload to GitHub",
  "Homepage.cloudBox.title": "Deploy to Strapi Cloud",
  "Homepage.cloudBox.subTitle": "Enjoy a Strapi-optimized stack including database, email provider, and CDN.",
  "Homepage.cloudBox.buttonText": "Deploy to Strapi Cloud",
  "Homepage.textBox.label.versioned": "Try Strapi Cloud for Free!",
  "Homepage.textBox.label.not-versioned": "Why uploading my project to GitHub?",
  "Homepage.textBox.text.versioned": "Strapi Cloud offers a 14 days free trial for you to experiment with your project on the cloud including all features.",
  "Homepage.textBox.text.not-versioned": "Strapi Cloud will fetch and deploy your project from your GitHub repository. This is the best way to version, manage and deploy your project. Follow the steps on GitHub to successfully upload it."
}
