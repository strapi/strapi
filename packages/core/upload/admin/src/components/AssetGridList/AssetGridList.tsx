// TODO: find a better naming convention for the file that was an index file before
import { Box, Grid, KeyboardNavigable, Typography } from '@strapi/design-system';
import styled from 'styled-components';

import { AssetCard } from '../AssetCard/AssetCard';

import { Draggable } from './Draggable';

import type { File } from '../../../../shared/contracts/files';
import type { AllowedTypes } from '../AssetCard/AssetCard';

const DraggableAssetCard = styled(AssetCard)`
  && {
    cursor: inherit;
  }
`;

export interface AssetGridListProps {
  allowedTypes?: AllowedTypes[];
  assets: File[];
  onEditAsset?: (asset: File) => void;
  onSelectAsset: (asset: File) => void;
  selectedAssets: File[];
  size?: 'S' | 'M';
  onReorderAsset?: (fromIndex: number, toIndex: number) => void;
  title?: string | null;
}

export const AssetGridList = ({
  allowedTypes = ['files', 'images', 'videos', 'audios'],
  assets,
  onEditAsset,
  onSelectAsset,
  selectedAssets,
  size = 'M',
  onReorderAsset,
  title = null,
}: AssetGridListProps) => {
  return (
    <KeyboardNavigable tagName="article">
      {title && (
        <Box paddingTop={2} paddingBottom={2}>
          <Typography tag="h2" variant="delta" fontWeight="semiBold">
            {title}
          </Typography>
        </Box>
      )}

      <Grid.Root gap={4}>
        {assets.map((asset, index) => {
          const isSelected = !!selectedAssets.find((currentAsset) => currentAsset.id === asset.id);

          if (onReorderAsset) {
            return (
              <Grid.Item key={asset.id} col={3} height="100%">
                <Draggable index={index} moveItem={onReorderAsset} id={asset.id}>
                  <DraggableAssetCard
                    allowedTypes={allowedTypes}
                    asset={asset}
                    isSelected={isSelected}
                    onEdit={onEditAsset ? () => onEditAsset(asset) : undefined}
                    onSelect={() => onSelectAsset(asset)}
                    size={size}
                  />
                </Draggable>
              </Grid.Item>
            );
          }

          return (
            <Grid.Item col={3} key={asset.id} height="100%" direction="column" alignItems="stretch">
              <AssetCard
                key={asset.id}
                allowedTypes={allowedTypes}
                asset={asset}
                isSelected={isSelected}
                onEdit={onEditAsset ? () => onEditAsset(asset) : undefined}
                onSelect={() => onSelectAsset(asset)}
                size={size}
              />
            </Grid.Item>
          );
        })}
      </Grid.Root>
    </KeyboardNavigable>
  );
};
