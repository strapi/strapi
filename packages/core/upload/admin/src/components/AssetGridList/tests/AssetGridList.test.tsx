import { DesignSystemProvider } from '@strapi/design-system';
import { render as renderTL } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

import en from '../../../translations/en.json';
import { AssetGridList } from '../AssetGridList';

type Messages = typeof en;

jest.mock('../../../utils', () => ({
  ...jest.requireActual('../../../utils'),
  getTrad: (x: string) => x,
}));

jest.mock('react-intl', () => ({
  useIntl: () => ({ formatMessage: jest.fn(({ id }: { id: keyof Messages }) => en[id]) }),
}));

const data = [
  {
    id: 1,
    name: 'strapi-cover_1fabc982ce.png',
    alternativeText: '',
    caption: '',
    width: 1066,
    height: 551,
    formats: {
      thumbnail: {
        name: 'thumbnail_strapi-cover_1fabc982ce.png',
        hash: 'thumbnail_strapi_cover_1fabc982ce_5b43615ed5',
        ext: '.png',
        mime: 'image/png',
        width: 245,
        height: 127,
        size: 3.37,
        path: null,
        url: '/uploads/thumbnail_strapi_cover_1fabc982ce_5b43615ed5.png',
      },
      large: {
        name: 'large_strapi-cover_1fabc982ce.png',
        hash: 'large_strapi_cover_1fabc982ce_5b43615ed5',
        ext: '.png',
        mime: 'image/png',
        width: 1000,
        height: 517,
        size: 22.43,
        path: null,
        url: '/uploads/large_strapi_cover_1fabc982ce_5b43615ed5.png',
      },
      medium: {
        name: 'medium_strapi-cover_1fabc982ce.png',
        hash: 'medium_strapi_cover_1fabc982ce_5b43615ed5',
        ext: '.png',
        mime: 'image/png',
        width: 750,
        height: 388,
        size: 14.62,
        path: null,
        url: '/uploads/medium_strapi_cover_1fabc982ce_5b43615ed5.png',
      },
      small: {
        name: 'small_strapi-cover_1fabc982ce.png',
        hash: 'small_strapi_cover_1fabc982ce_5b43615ed5',
        ext: '.png',
        mime: 'image/png',
        width: 500,
        height: 258,
        size: 8.38,
        path: null,
        url: '/uploads/small_strapi_cover_1fabc982ce_5b43615ed5.png',
      },
    },
    hash: 'strapi_cover_1fabc982ce_5b43615ed5',
    ext: '.png',
    mime: 'image/png',
    size: 6.85,
    url: '/uploads/strapi_cover_1fabc982ce_5b43615ed5.png',
    previewUrl: null,
    provider: 'local',
    provider_metadata: null,
    createdAt: '2021-09-14T07:32:50.816Z',
    updatedAt: '2021-09-14T07:32:50.816Z',
  },
  {
    id: 5,
    name: 'mov_bbb.mp4',
    alternativeText: '',
    caption: '',
    width: null,
    height: null,
    formats: null,
    hash: 'mov_bbb_2f3907f7aa',
    ext: '.mp4',
    mime: 'video/mp4',
    size: 788.49,
    url: '/uploads/mov_bbb_2f3907f7aa.mp4',
    previewUrl: null,
    provider: 'local',
    provider_metadata: null,
    createdAt: '2021-09-14T07:48:30.882Z',
    updatedAt: '2021-09-14T07:48:30.882Z',
  },
  {
    id: 6,
    name: 'CARTE MARIAGE AVS - Printemps.pdf',
    alternativeText: '',
    caption: '',
    width: null,
    height: null,
    formats: null,
    hash: 'CARTE_MARIAGE_AVS_Printemps_1f87b19e18',
    ext: '.pdf',
    mime: 'application/pdf',
    size: 422.37,
    url: '/uploads/CARTE_MARIAGE_AVS_Printemps_1f87b19e18.pdf',
    previewUrl: null,
    provider: 'local',
    provider_metadata: null,
    createdAt: '2021-09-14T07:51:59.845Z',
    updatedAt: '2021-09-14T07:51:59.845Z',
  },
];

const setup = (props = { assets: data, selectedAssets: [], onSelectAsset: jest.fn() }) =>
  renderTL(
    <MemoryRouter>
      <DesignSystemProvider>
        <AssetGridList {...props} />
      </DesignSystemProvider>
    </MemoryRouter>
  );

describe('MediaLibrary / AssetList', () => {
  beforeAll(() => {
    // see https://github.com/testing-library/react-testing-library/issues/470
    Object.defineProperty(HTMLMediaElement.prototype, 'muted', {
      set() {},
    });
  });

  it('snapshots the asset list', () => {
    const { container } = setup();

    expect(container).toMatchSnapshot();
  });
});
