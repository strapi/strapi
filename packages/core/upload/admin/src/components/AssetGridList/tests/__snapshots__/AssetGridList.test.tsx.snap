// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaLibrary / AssetList snapshots the asset list 1`] = `
.c1 {
  height: 100%;
}

.c4 {
  height: 100%;
  background: #ffffff;
  border-radius: 4px;
  border-style: solid;
  border-width: 1px;
  border-color: #eaeaef;
  box-shadow: 0px 1px 4px rgba(33, 33, 52, 0.1);
}

.c6 {
  position: relative;
}

.c15 {
  padding-block-start: 8px;
  padding-block-end: 8px;
  padding-inline-start: 12px;
  padding-inline-end: 12px;
}

.c18 {
  padding-block-start: 4px;
}

.c23 {
  padding-block-start: 4px;
  flex-grow: 1;
}

.c26 {
  padding-inline-start: 8px;
  padding-inline-end: 8px;
  min-width: 20px;
  background: #eaeaef;
}

.c33 {
  padding-block-start: 4px;
  padding-inline-end: 4px;
  padding-block-end: 4px;
  padding-inline-start: 4px;
  bottom: 4px;
  right: 4px;
  color: #ffffff;
  background: #32324d;
  border-radius: 4px;
  position: absolute;
}

.c35 {
  width: 100%;
  height: 16.4rem;
}

.c2 {
  align-items: stretch;
  flex-direction: column;
  display: flex;
}

.c7 {
  align-items: center;
  justify-content: center;
  flex-direction: row;
  display: flex;
}

.c9 {
  gap: 8px;
  align-items: center;
  flex-direction: row;
  display: flex;
}

.c16 {
  align-items: flex-start;
  flex-direction: row;
  display: flex;
}

.c24 {
  align-items: center;
  flex-direction: row;
  display: flex;
}

.c27 {
  align-items: center;
  justify-content: center;
  flex-direction: row;
  display: inline-flex;
}

.c37 {
  gap: 8px;
  align-items: center;
  flex-direction: column;
  display: flex;
}

.c19 {
  font-size: 1.4rem;
  line-height: 1.43;
  color: currentcolor;
}

.c20 {
  font-size: 1.2rem;
  line-height: 1.33;
  color: #32324d;
  font-weight: 600;
}

.c21 {
  font-size: 1.2rem;
  line-height: 1.33;
  color: #666687;
}

.c30 {
  font-weight: 600;
  font-size: 1.1rem;
  line-height: 1.45;
  text-transform: uppercase;
  color: #666687;
  line-height: 1rem;
}

.c34 {
  font-size: 1.2rem;
  line-height: 1.33;
  color: #ffffff;
}

.c38 {
  font-size: 1.2rem;
  line-height: 1.33;
  color: #8e8ea9;
}

.c32 {
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

.c28 {
  border-radius: 4px;
  padding-block: 0.7rem;
}

.c10 {
  position: absolute;
  top: 12px;
  left: 12px;
}

.c14 {
  margin: 0;
  padding: 0;
  max-height: 100%;
  max-width: 100%;
  object-fit: contain;
}

.c13 {
  display: flex;
  justify-content: center;
  height: 16.4rem;
  width: 100%;
  background: repeating-conic-gradient(#f6f6f9 0% 25%, transparent 0% 50%) 50%/20px 20px;
  border-top-left-radius: 4px;
  border-top-right-radius: 4px;
}

.c25 {
  margin-left: auto;
  flex-shrink: 0;
}

.c29 {
  margin-left: 4px;
}

.c11 {
  background: #ffffff;
  width: 2rem;
  height: 2rem;
  border-radius: 4px;
  border: 1px solid #c0c0cf;
  position: relative;
  z-index: 0;
  display: flex;
  justify-content: center;
  align-items: center;
  flex: 0 0 2rem;
}

.c11[data-state='checked'],
.c11[data-state='indeterminate'] {
  border: 1px solid #4945ff;
  background-color: #4945ff;
}

.c11[data-disabled] {
  background-color: #dcdce4;
}

.c11::before {
  content: '';
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
  width: 100%;
  height: 100%;
  z-index: -1;
  min-width: 44px;
  min-height: 44px;
}

.c12 {
  width: 100%;
  height: 100%;
  cursor: pointer;
}

.c17 {
  word-break: break-all;
}

.c8 {
  border-bottom: 1px solid #eaeaef;
}

.c0 {
  display: grid;
  grid-template-columns: repeat(12, 1fr);
  gap: 16px;
}

.c3 {
  grid-column: span 12;
  max-width: 100%;
}

.c22 {
  text-transform: uppercase;
}

.c5 {
  cursor: pointer;
}

.c36 {
  border-radius: 4px 4px 0 0;
  background: linear-gradient(     180deg,     #ffffff 0%,     #f6f6f9 121.48%   );
}

.c31 canvas,
.c31 video {
  display: block;
  pointer-events: none;
  max-width: 100%;
  max-height: 16.4rem;
}

@media (min-width: 520px) {
  .c3 {
    grid-column: span 12;
  }
}

@media (min-width: 768px) {
  .c3 {
    grid-column: span 12;
  }
}

@media (min-width: 1080px) {
  .c3 {
    grid-column: span 3;
  }
}

<div>
  <div
    class=""
  >
    <div
      class="c0"
    >
      <div
        class="c1 c2 c3"
      >
        <article
          aria-labelledby=":r0:-title"
          class="c4 c5"
          role="button"
          tabindex="-1"
        >
          <div
            class="c6 c7 c8"
          >
            <div>
              <div
                class="c9 c10"
              >
                <button
                  aria-checked="false"
                  aria-labelledby=":r0:-title"
                  class="c11"
                  data-state="unchecked"
                  role="checkbox"
                  type="button"
                  value="on"
                >
                  <span
                    class="c12"
                    data-state="unchecked"
                    style="pointer-events: auto; display: inline-flex;"
                  />
                </button>
              </div>
            </div>
            <div
              class="c13"
            >
              <img
                alt="strapi-cover_1fabc982ce.png"
                aria-hidden="true"
                class="c14"
                src="http://localhost:1337/uploads/thumbnail_strapi_cover_1fabc982ce_5b43615ed5.png?updatedAt=2021-09-14T07%3A32%3A50.816Z"
              />
            </div>
          </div>
          <div
            class="c15"
          >
            <div
              class="c16"
            >
              <div
                class="c17"
              >
                <div
                  class="c18"
                >
                  <h2
                    class="c19"
                  >
                    <span
                      class="c20"
                      id=":r0:-title"
                    >
                      strapi-cover_1fabc982ce.png
                    </span>
                  </h2>
                </div>
                <div
                  class="c21"
                >
                  <span
                    class="c22"
                  >
                    png
                  </span>
                   - 1066âœ•551
                </div>
              </div>
              <div
                class="c23 c24"
              >
                <div
                  class="c25"
                >
                  <div
                    class="c26 c27 c28 c29"
                  >
                    <span
                      class="c30"
                    >
                      Image
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </article>
      </div>
      <div
        class="c1 c2 c3"
      >
        <article
          aria-labelledby=":r2:-title"
          class="c4 c5"
          role="button"
          tabindex="-1"
        >
          <div
            class="c6 c7 c8"
          >
            <div>
              <div
                class="c9 c10"
              >
                <button
                  aria-checked="false"
                  aria-labelledby=":r2:-title"
                  class="c11"
                  data-state="unchecked"
                  role="checkbox"
                  type="button"
                  value="on"
                >
                  <span
                    class="c12"
                    data-state="unchecked"
                    style="pointer-events: auto; display: inline-flex;"
                  />
                </button>
              </div>
            </div>
            <div
              class="c13"
            >
              <div
                class="c31"
              >
                <figure
                  class=""
                >
                  <video
                    crossorigin="anonymous"
                    src="http://localhost:1337/uploads/mov_bbb_2f3907f7aa.mp4"
                  >
                    <source
                      type="video/mp4"
                    />
                  </video>
                  <figcaption
                    class="c32"
                  >
                    mov_bbb.mp4
                  </figcaption>
                </figure>
              </div>
            </div>
            <time
              class="c33"
            >
              <span
                class="c34"
              >
                ...
              </span>
            </time>
          </div>
          <div
            class="c15"
          >
            <div
              class="c16"
            >
              <div
                class="c17"
              >
                <div
                  class="c18"
                >
                  <h2
                    class="c19"
                  >
                    <span
                      class="c20"
                      id=":r2:-title"
                    >
                      mov_bbb.mp4
                    </span>
                  </h2>
                </div>
                <div
                  class="c21"
                >
                  <span
                    class="c22"
                  >
                    mp4
                  </span>
                </div>
              </div>
              <div
                class="c23 c24"
              >
                <div
                  class="c25"
                >
                  <div
                    class="c26 c27 c28 c29"
                  >
                    <span
                      class="c30"
                    >
                      Video
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </article>
      </div>
      <div
        class="c1 c2 c3"
      >
        <article
          aria-labelledby=":r4:-title"
          class="c4 c5"
          role="button"
          tabindex="-1"
        >
          <div
            class="c6 c7 c8"
          >
            <div>
              <div
                class="c9 c10"
              >
                <button
                  aria-checked="false"
                  aria-labelledby=":r4:-title"
                  class="c11"
                  data-state="unchecked"
                  role="checkbox"
                  type="button"
                  value="on"
                >
                  <span
                    class="c12"
                    data-state="unchecked"
                    style="pointer-events: auto; display: inline-flex;"
                  />
                </button>
              </div>
            </div>
            <div
              class="c35 c7 c36"
            >
              <div
                class="c37"
              >
                <svg
                  aria-label="CARTE MARIAGE AVS - Printemps.pdf"
                  fill="#8e8ea9"
                  height="24"
                  viewBox="0 0 32 32"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M6 15h20a1 1 0 0 0 1-1v-3a1 1 0 0 0-.293-.707l-7-7A1 1 0 0 0 19 3H7a2 2 0 0 0-2 2v9a1 1 0 0 0 1 1m13-9.5 5.5 5.5H19zM28 19a1 1 0 0 1-1 1h-3v2h2a1 1 0 0 1 0 2h-2v2a1 1 0 0 1-2 0v-7a1 1 0 0 1 1-1h4a1 1 0 0 1 1 1M8 18H6a1 1 0 0 0-1 1v7a1 1 0 1 0 2 0v-1h1a3.5 3.5 0 1 0 0-7m0 5H7v-3h1a1.5 1.5 0 1 1 0 3m8-5h-2a1 1 0 0 0-1 1v7a1 1 0 0 0 1 1h2a4.5 4.5 0 1 0 0-9m0 7h-1v-5h1a2.5 2.5 0 0 1 0 5"
                  />
                </svg>
                <span
                  class="c38"
                />
              </div>
            </div>
          </div>
          <div
            class="c15"
          >
            <div
              class="c16"
            >
              <div
                class="c17"
              >
                <div
                  class="c18"
                >
                  <h2
                    class="c19"
                  >
                    <span
                      class="c20"
                      id=":r4:-title"
                    >
                      CARTE MARIAGE AVS - Printemps.pdf
                    </span>
                  </h2>
                </div>
                <div
                  class="c21"
                >
                  <span
                    class="c22"
                  >
                    pdf
                  </span>
                </div>
              </div>
              <div
                class="c23 c24"
              >
                <div
                  class="c25"
                >
                  <div
                    class="c26 c27 c28 c29"
                  >
                    <span
                      class="c30"
                    >
                      Doc
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </article>
      </div>
    </div>
  </div>
  <span
    class="c32"
  >
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-log"
      role="log"
    />
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-status"
      role="status"
    />
    <p
      aria-live="assertive"
      aria-relevant="all"
      id="live-region-alert"
      role="alert"
    />
  </span>
</div>
`;
